test/stSolidityTestFiller.json

```diff
@@ -489,6 +461,23 @@
         {
             "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
                 "balance" : "100000",
+		"//" : "contract contract1		",
+		"//" : "{				",
+		"//" : "    uint value;			",
+		"//" : "    function run()		",
+		"//" : "    {				",
+		"//" : "        value = 225;		",
+		"//" : "    }				",
+		"//" : "}				",
+		"//" : "				",
+		"//" : "contract contract2		",
+		"//" : "{				",
+		"//" : "    uint value2;		",
+		"//" : "    function run()		",
+		"//" : "    {				",
+		"//" : "        value2 = 335;		",
+		"//" : "    }				",
+		"//" : "}				",
                 "code" : "0x60003560e060020a90048063c040622614601557005b601b6021565b60006000f35b61014f60008190555056",
```

Comment:
When you have two contracts, which code did you put in?  And does the other contract in any way affect the "code"